Makefile

```diff
@@ -121,3 +121,13 @@ terratest/install: | guard/program/go guard/program/dep
 
 terratest/test: | guard/program/go guard/program/dep
 	cd tests && go test -timeout 20m
+
+clean:
+	rm -rf vendor
+
+VENDOR ?= vendor/github.com/awslabs/aws-config-rules
+vendor: $(VENDOR)
+
+vendor/%:
```

Comment:
I guess the answer is 'just don't forget to run it manually before a release' but would it make sense to include this as part of the release workflow?

Reply:
not so much _every_ release, as just periodically, when there are new or updated config rules we want to use... then it's just: `make clean && make vendor`. worth putting in the README though

Reply:
Added to the README